app/controllers/ExplainerReindexController.scala

```diff
@@ -0,0 +1,79 @@
+package controllers
+
+import com.gu.atom.data.DynamoDataStore
+import com.gu.atom.publish.{AtomReindexer, PreviewAtomReindexer, PublishedAtomReindexer}
+import db.ExplainerDBAPI
+import play.api.Configuration
+import play.api.libs.ws.WSClient
+import play.api.mvc.{ Action, ActionBuilder, AnyContent, Controller, Result, Request }
+import com.gu.contentatom.thrift.{ContentAtomEvent, EventType}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{ Failure, Success, Try }
+import play.api.libs.json.Json
+import play.api.Logger
+
+class ExplainerReindexController(
+  val wsClient: WSClient,
+  explainerDB: ExplainerDBAPI,
+  previewDataStore: DynamoDataStore,
+  publishedDataStore: DynamoDataStore,
+  previewReindexer: PreviewAtomReindexer,
+  publishedReindexer: PublishedAtomReindexer,
+  config: Configuration
+)(implicit ec: ExecutionContext) extends Controller with PanDomainAuthActions {
+
+  var lastPublished: Int = 0
```

Comment:
Will these guys get set back to zero on completion by the library?

Reply:
Nope, that's not necessary (the reindex controller for regular atoms does not either)